lib/trailblazer/operation.rb

```diff
@@ -29,8 +29,8 @@ module Trailblazer
   class Operation
     module FastTrackActivity
       builder_options = {
-        track_end:     Railway::End::Success.new(semantic: :success),
-        failure_end:   Railway::End::Failure.new(semantic: :failure),
+        track_end: Railway::End::Success.new(semantic: :success),
+        failure_end: Railway::End::Failure.new(semantic: :failure),
```

Comment:
Not a fan of this, looks messy.

Reply:
The setting for rubocop to stick to the other style of formatting (i.e. align on colon - which I also prefer) is this:

```yaml
Layout/AlignHash:
  EnforcedColonStyle: separator
```

Reply:
@konung :+1: thanks